tests/test_lesion_froc.py

```diff
@@ -0,0 +1,320 @@
+import os
+import unittest
+from unittest import skipUnless
+
+import numpy as np
+from parameterized import parameterized
+
+from monai.apps.pathology.metrics import LesionFROC
+from monai.utils import optional_import
+
+_, has_cucim = optional_import("cucim")
+_, has_skimage = optional_import("skimage.measure")
+_, has_sp = optional_import("scipy.ndimage")
+PILImage, has_pil = optional_import("PIL.Image")
+
+
+def save_as_tif(filename, array):
+    array = array[::-1, ...]  # Upside-down
+    img = PILImage.fromarray(array)
+    if not filename.endswith(".tif"):
+        filename += ".tif"
+    img.save(os.path.join("tests", "testing_data", filename))
```

Comment:
please  update `.gitignore` for these new files

Reply:
I prepended the images with `temp_` and add `tests/testing_data/temp_*` to `.gitignore`